### PR TITLE
add percent text

### PR DESCRIPTION
**Pull request by roman-webtex**

### CODE REVIEW

File: unote [hunk 1]
```diff
@@ -23,6 +23,13 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
 # OR OTHER DEALINGS IN THE SOFTWARE.
 
+# (c) 2020 Roman Dmytrenko <roman.webtex@gmail.com>
```
Comment (michurin) [inline unote]: Please move it upper to line 4. Like this
```
...
# Copyright (c) 1998-2017 Alexey Michurin <a.michurin@gmail.com>
# Copyright (c) 2020 Roman Dmytrenko <roman.webtex@gmail.com>
#
# Permission is hereby granted, free of charge, to any person
...
```

File: unote [hunk 2]
```diff
@@ -23,6 +23,13 @@
...
+#
+# Add percent text for bar
+#
+# add 'tc' parameter for percent text color
+#
```
Comment (michurin) [inline unote]: Let's remove "changelog" from here

File: unote [hunk 3]
```diff
@@ -37,7 +44,11 @@ proc create_label {create_new element text_opts font_opts} {
     font configure [$element cget -font] {*}$font_opts
 }
 
-proc create_bar {create_new element w h b c1 c2 c3 percent} {
+
+# add $c4 - percent text color
+# add $font_opts
```
Comment (michurin) [inline unote]: Well, as long as you started to comment code, it would be great to add description for all args

File: unote [hunk 4]
```diff
@@ -220,6 +233,7 @@ proc notmalize_message {text_opts body} {
     #   fg
     #   bg
     #   bd
+    #   tc - new parameter for percent text color
```
Comment (michurin) [inline unote]: Let's add descriptions to all the rest options.
And remove the "new" please. As well as "parameter for", it is obvious.

File: unote [hunk 5]
```diff
@@ -301,8 +315,18 @@ proc notmalize_message {text_opts body} {
...
         }]
+
+        # add font config for create_bar
+        set font_opts [simple_named_options $opts {
+            -size size 16
+            -family family Helvetica
+        }]
+        dict set font_opts -weight [dict_get_flag $opts bold bold normal]
+        dict set options {font} $font_opts
+
```
Comment (michurin) [inline unote]: Please remove leading spaces in this line.

File: unote [hunk 6]
```diff
@@ -301,8 +315,18 @@ proc notmalize_message {text_opts body} {
             bd #009999
             bg #000000
             fg #00ffff
+            tc #ffff00
             percent 50
         }]
+
+        # add font config for create_bar
```
Comment (michurin) [inline unote]: Useless comment

**Review by michurin — COMMENTED** [review]
Please add description of new options to `README.md`.

**Review by michurin — APPROVED** [review]
:+1:

**Comment by roman-webtex** [comment]
спасибо, давно хотел что-то похожее и совершенно случайно наткнулся на ваш
сайт.

**Pull request merged**